apps/web/src/pages/Pools/PoolAdvertisementPage/components/LanguageRequirementDialog.tsx

```diff
@@ -0,0 +1,469 @@
+  headingLevel,
+}: LanguageRequirementDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+
+  const languageProfilesList = [
+    {
+      title: languageProfiles.frenchEssential,
+      description: languageProfileDescriptions.frenchEssential,
+    },
+    {
+      title: languageProfiles.englishEssential,
+      description: languageProfileDescriptions.englishEssential,
+    },
+    {
+      title: languageProfiles.bilingualIntermediate,
+      description: languageProfileDescriptions.bilingualIntermediate,
+    },
+    {
+      title: languageProfiles.bilingualAdvanced,
+      description: languageProfileDescriptions.bilingualAdvanced,
+    },
+    {
+      title: languageProfiles.various,
+      description: languageProfileDescriptions.various,
+    },
+  ];
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <Button
+          mode="icon_only"
+          color="secondary"
+          icon={InformationCircleIcon}
+          aria-label={intl.formatMessage({
+            defaultMessage: "Learn about how language requirements work.",
+            id: "WhHVMt",
+            description:
+              "Info button label for pool language requirements details.",
+          })}
+        />
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header>
+          {intl.formatMessage({
+            defaultMessage:
+              "Language requirements and second language proficiency",
+            id: "+7yxK0",
+            description: "Heading for the pool language requirements dialog",
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          <div data-h2-display="base(grid)" data-h2-gap="base(x1)">
+            <div data-h2-display="base(grid)" data-h2-gap="base(x1)">
+              <div>
+                <Heading
+                  level={headingLevel}
+                  size="h6"
+                  data-h2-margin="base(0, 0, x.25, 0)"
+                >
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Language requirements on GC Digital Talent",
+                    id: "h1UJ1Q",
+                    description: "Sub-heading for language requirements dialog",
+                  })}
+                </Heading>
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "These are the language profiles that you can find on GC Digital Talent",
+                    id: "sry9Wl",
+                    description:
+                      "Language profiles list for language requirements dialog",
+                  }) + intl.formatMessage(commonMessages.dividingColon)}
+                </p>
+              </div>
+              <ul data-h2-display="base(grid)" data-h2-gap="base(x.5)">
+                {languageProfilesList.map((item) => (
+                  <li key={uniqueId()}>
+                    <p
+                      data-h2-color="base(primary)"
```

Comment:
This is too dark for dark mode. I think we usually use `primary.dark` or something :thinking:

Reply:
Okay :+1: Fixed by [50507dc](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10628/commits/50507dc5502e42c30074e71fa1677c6b602c86c6)